### PR TITLE
MUHU-179:added functionality to add new card if presale, insale og ev…

### PR DESCRIPTION
**Pull request by lilosti**
…entdate is changed

### CODE REVIEW

File: src/Asana/AsanaEventManager.php [hunk 1]
```diff
@@ -50,38 +50,38 @@ public function createCards(): void
...
+                    $this->asanaApiClient->createCardNewEventsBoard($eventData);
+                    $this->cardCreated($eventsNewEvent['id'], self::EVENTS);
+                }
+
+                foreach ($eventsNewEventsExternal as $eventsNewEventExternal) {
+                    $eventData = $this->getEventData($this->eventRepository->find($eventsNewEventExternal['id']));
+                    $this->asanaApiClient->createCardNewEventsGratisandExternBoard($eventData);
+                    $this->cardCreated($eventsNewEventExternal['id'], self::EVENTS_EXTERN);
+                }
+        */
```
Comment (rimi-itk) [inline src/Asana/AsanaEventManager.php]: Remove this code if not used.

File: src/Asana/AsanaEventManager.php [hunk 2]
```diff
@@ -145,6 +149,11 @@ private function cardCreated(string $id, string $type)
                 break;
             case self::CALENDAR_EVENTS:
                 $card->setCreatedInCalendar(true);
+                $yesplanevent = $this->eventRepository->find($id);
+                $yesplanevent->setInSaleDateUpdated(false);
+                $yesplanevent->setEventDateUpdated(false);
+                $yesplanevent->setInPresaleDateUpdated(false);
+                $yesplanevent->setIsNewEvent(false);
```
Comment (rimi-itk) [inline src/Asana/AsanaEventManager.php]: Will lead to null dereference errors if the event is not found. Do you know for sure that it always exists?

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: src/Asana/AsanaEventManager.php (hunk 1 above)
Comment (lilosti) [reply src/Asana/AsanaEventManager.php]: not supposed be commented out...

File: src/Asana/AsanaEventManager.php (hunk 2 above)
Comment (lilosti) [reply src/Asana/AsanaEventManager.php]: You never know for sure..

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**